versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 2.0"
+      version = ">= 4.21.0"
```

Comment:
The new s3 module also requires the terraform 1.x and awsutils provider

Reply:
Also the example/complete versions.tf should match the root versions tf

Reply:
Thank you very much @nitrocode! I did the the changes you pointed out, and the misses I had. 
I still haven't added the awsutils provider as a requirement in this module because it was already in the s3 module, and when I do an init without the awsutils in codebuild it is being downloaded as expected. 
<img width="633" alt="image" src="https://user-images.githubusercontent.com/37238646/178282536-d1c0ddf4-3519-4e8e-964d-2c4617e66d94.png">
Is that ok?

Reply:
Hmmm perhaps it's fine then.